Doc/library/multiprocessing.rst

```diff
@@ -1676,7 +1676,7 @@ Manager processes will be shutdown as soon as they are garbage collected or
 their parent process exits.  The manager classes are defined in the
 :mod:`multiprocessing.managers` module:
 
-.. class:: BaseManager([address[, authkey]])
+.. class:: BaseManager(address=None, authkey=None, serializer='pickle', ctx=None, *, shutdown_timeout=1.0)
```

Comment:
It's a bit weird to mention `ctx` and `serializer` here without describing their purpose below. Just keep them undocumented (and/or create a separate issue for that)?

Reply:
I completed the doc.